ubtue_Brill.js

```diff
@@ -70,13 +70,21 @@ function getSearchResults(doc) {
 function postProcess(doc, item) {
 	let title = ZU.xpathText(doc, '//meta[@name="citation_title"]//@content');
 	if (title) item.title = title; 
-	if (!item.abstractNote) {
-	  item.abstractNote = ZU.xpath(doc, '//section[@class="abstract"]//p');
-	  if (item.abstractNote && item.abstractNote.length > 0)
-		 item.abstractNote = item.abstractNote[0].textContent.trim();
-	  else
-		 item.abstractNote = '';
+	let abstracts = ZU.xpath(doc, '//section[@class="abstract"]//p');
+	//multiple abstracts
+	if (abstracts && abstracts.length > 0) {
+		item.abstractNote = abstracts[0].textContent.trim().replace(/^(Résumé\s?)/i, "");
+		if (abstracts.length ) {
+			for (let abs of abstracts) {
```

Comment:
The first abstract is added and afterwards compared again as to whether or not to add it again. It is probably better to slice the original `abstracts` structure after the first assignment (c.f. e.g. `https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/slice`) and then add the rest unconditionally.

Reply:
the abstracts.length is "4" with twice the same objects value. `Z.debug(abstracts.length)` Therefore I have to do this unusual if-comparison for deduplication.

Reply:
Think this deduplication should be made explicit then:
````js
    let abstracts = ZU.xpath(doc, '//section[@class="abstract"]//p');
    //multiple abstracts
    if (abstracts && abstracts.length > 0) {
	// Deduplicate original results
	abstracts = [...new Set(abstracts.map(abstract => abstract.textContent))];
	item.abstractNote = abstracts[0].trim().replace(/^(Résumé\s?)/i, "");
	if (abstracts.length > 1) {
	    for (let abs of abstracts.slice(1)) {
                  item.notes.push({
	 	         note: "abs:" + ZU.trimInternal(abs),
	 	  });
            }
	}
    }

Reply:
ok. I didn't come up with that idea, but the one line code with arrow function looks good :+1: For my understanding
```
//arrow function
abstracts = [...new Set(abstracts.map(abstract => abstract.textContent))];

//regular function
abstracts = Array.from(new Set(abstracts.map(function(abstract) {
	return abstract.textContent
})));
```

Reply:
These two function should yield identical results. The first one uses the spread operator `...` and an arrow notation as argument to map, so the first one is a "spread and arrow function" ;-).

Reply:
Unfortunately, I probably cannot commit to your repository, so I cannot fix the formatting problems myself. I will merge it now but time permitting please fix them.